### PR TITLE
support to config the version white list

### PR DESCRIPTION
**Pull request by Nicole00**
close https://github.com/vesoft-inc/nebula-go/issues/299

### CODE REVIEW

**Comment by HarrisChu** [comment]
why need user provide the version?
user changes the `client_white_list` in graphd?

File: connection.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ type connection struct {
 	sslConfig    *tls.Config
 	useHTTP2     bool
 	httpHeader   http.Header
+	version 	 string
```
Comment (HarrisChu) [inline connection.go:33]: please run `go fmt` to make aligned

Comment (Nicole00) [reply connection.go]: ok

**Comment by Nicole00** [comment]
> why need user provide the version? user changes the `client_white_list` in graphd?

Yeah, this `version` config is optional. the default value is `3.0.0`, and if you set value for `version`, need to make sure the value is in  `client_white_list` in `nebula-graph.conf`.

**Pull request merged**